app/wizards/placements/add_placement_wizard.rb

```diff
@@ -33,6 +33,7 @@ def create_placement
 
     def build_placement
       placement = school.placements.build
+      placement.academic_year = ::Placements::AcademicYear.current
```

Comment:
Can we add a comment that this will change

Reply:
Due to the fact that this enables placements to be created with the same functionality they had before. When this changes it is a new feature so there is no reason to call this out.